pkg/unalias/unalias.go

```diff
@@ -0,0 +1,27 @@
+package unalias
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tfversion/tfversion/pkg/alias"
+	"github.com/tfversion/tfversion/pkg/helpers"
+)
+
+// UnaliasVersion removes the symlink for the specified alias.
+func UnaliasVersion(aliasName string) {
```

Comment:
Would RmAlias be more appropriate perhaps? Since `version` relates to our install flow